### ISSUE TITLE
Rate limit

### ISSUE THREAD

**Issue by Johannes-Krabbe** [malikpiara/cuiff#35]
This needs to be discussed:

Rate limit for creating posts (or boards) to prevent spam creating

**Comment by malikpiara** [comment]
Good point! @dewil-official mentioned this a while ago but it slipped my mind.
